projections_read_models_experiment/projector.py

```diff
@@ -0,0 +1,58 @@
+from esdbclient.exceptions import DeadlineExceeded
+
+from event_sourcery.event_store import EventStore, Position, Recorded
+from projections_read_models_experiment.logger import logger
+from projections_read_models_experiment.projection import Projection, ProjectionName
+from projections_read_models_experiment.subscription import Subscription
+
+
+class Projector:
+    def __init__(self, event_store: EventStore) -> None:
+        self._event_store = event_store
+
+        # I believe it should be backed by same mechanism as StorageStrategy
+        # I bet EventStoreDB have this built-in, for other backends
+        # we'd have to store it in respective DB
+        self._projections_positions: dict[ProjectionName, Position] = {}
```

Comment:
Projector can't hold it BECAUSE it should be kept in the same storage as projection